dex/testing/dcrdex/harness.sh

```diff
@@ -73,6 +73,11 @@ adminsrvpass=adminpass
 adminsrvaddr=127.0.0.1:16542
 EOF
 
+# Set the postgres user pass if provided. 
+if ! [ -z ${PG_PASS} ]; then
```

Comment:
If `-n` worked, that's ok too.  I just didn't think about the negation.  I also don't know what's posix compliant or not off hand, so whatever works for both bash and zsh in their posix compliant modes... (noting that [`-v` seems to exist in zsh](http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html#Conditional-Expressions) too).

Reply:
Oh, double quote to prevent globbing and word splitting (`if [ -n "${PG_PASS}" ]; then`).